src/ds3_uint64_string_map.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2017 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+#ifndef __DS3_UINT64_T_STRING_MAP__
+#define __DS3_UINT64_T_STRING_MAP__
+
+#include <stdlib.h>
+
+#include "stdint.h"
+#include "ds3_bool.h"
+#include "ds3_string.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// For windows DLL symbol exports.
+#ifdef _WIN32
```

Comment:
This appears to be duplicated again.  Is there a single place where we can define this and then include it?